src/com/epam/javatrainings/classdecomposition/Main.java

```diff
@@ -0,0 +1,192 @@
+package com.epam.javatrainings.classdecomposition;
+
+//import Pizza.Piiza;
+
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static Scanner cs = new Scanner(System.in);
+    public static ArrayList<Order> orders = new ArrayList();
+    public static ArrayList<Pizza> pizzas = new ArrayList();
+
+    public static void main(String[] args) {
+        boolean ok = true;
+        while (ok) {
+            print();
+            String x = cs.nextLine();
+            switch (x) {
+                case ("0"):
+                    ok = false;
+                    break;
+                case ("1"):
+                    addpizza();
+                    empty();
+                    break;
+                case ("2"):
+                    addorder();
+                    break;
+                case "3":
+                    showorder();
+                    break;
+                case "4":
+                    empty();
+                    break;
+                case "5":
+                    orders();
+                    break;
+
+                default:
+                    System.err.println("Error");
+                    break;
+
+            }
+        }
+
+
+    }
+
+    public static void print() {
```

Comment:
Please create a Order service class and locate all logic there, in main class you should only instantiate that class and use its method, do not make everything static, please more OOP.